types/tabulator-tables/tabulator-tables-tests.ts

```diff
@@ -1126,36 +1126,67 @@ table = new Tabulator('#test', {
 // 5.3 Testing ColumnDefinition.headerMenuIcon
 const headerMenuForIconTest: Array<MenuObject<ColumnComponent> | MenuSeparator> = [
     {
-        label:"Hide Column",
+        label: 'Hide Column',
         action(e, column) {
             column.hide();
-        }
+        },
     },
 ];
-let headerMenuIconElement = document.createElement("span");
-headerMenuIconElement.innerText = "Filter";
+let headerMenuIconElement = document.createElement('span');
+headerMenuIconElement.innerText = 'Filter';
 
 table = new Tabulator('#testHeaderMenuIcon', {
     columns: [
         {
             field: 'test_inline',
             title: 'Test inline',
             headerMenuIcon: "<i class='fas fa-filter'></i>",
-            headerMenu: headerMenuForIconTest
+            headerMenu: headerMenuForIconTest,
         },
         {
             field: 'test_element',
             title: 'Test DOM Element',
             headerMenuIcon: headerMenuIconElement,
-            headerMenu: headerMenuForIconTest
+            headerMenu: headerMenuForIconTest,
         },
         {
             field: 'test_function',
             title: 'Test function',
             headerMenuIcon(component) {
                 return "<i class='fas fa-filter'></i>";
             },
-            headerMenu: headerMenuForIconTest
-        }
-    ]
+            headerMenu: headerMenuForIconTest,
+        },
+    ],
 });
+
+// 5.4
+
+table = new Tabulator('#test', {
+    headerSortClickElement: 'icon',
+    groupDblClickPopup: 'Im a Popup',
```

Comment:
Maybe you want to fix the typo in all occurrences of this string? "I'm a Popup"...

Reply:
Eh, not really for the tests. Do you feel strongly about it?

Reply:
It's a simple change and makes it look better if written correctly. You can also use [Lorem Ipsum](https://en.wikipedia.org/wiki/Lorem_ipsum) for dummy text.

I don't feel strongly about that, but I look after the small things too. If you say "let's keep that" then so be it and I'll approve.

Reply:
It was copied from the website examples again. I kind of do want to fix it but don't want to spend the time this morning to bring it back up, I'm on mobile this morning . Would you want to push the update?